### PR TITLE
Delete routesteps mq 6

### PR DESCRIPTION
**Pull request by mitchell-mccracken**
Closes MQ-6

Solution: 
* Added content to `routeStep/controller.ts` file including `deleteRouteStep` function.
Added `routeStep` path to `app.ts`

### CODE REVIEW

**Review by jbolles94 — CHANGES_REQUESTED** [review]

File: src/routeStep/controller.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+module.exports = router;
+
+//routes
+router.delete('/', deleteRouteStep);
+
+
+
+//functions
+function deleteRouteStep(
```
Comment (jbolles94) [inline src/routeStep/controller.ts:15]: can we add a unit test for this?

File: src/routeStep/controller.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  req: { body: { routeStepId: string; }; }, 
+  res: any
+  ) {
+  ExpressHandler(
+    async () => {
+      const { routeStepId } = req.body;
+      if ( !routeStepId ) return HTTPError('no route step id provided');
```
[the remark this answers is not in the document]
Comment (jarrilla) [reply src/routeStep/controller.ts:22]: fixed

File: src/routeStep/controller.ts (hunk 1 above)
Comment (jarrilla) [reply src/routeStep/controller.ts]: done

**Comment by jarrilla** [comment]
tests passing; merging

**Pull request merged**